policies/openstack/CoreCAPI.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: capo-policy
+spec:
+  background: false
+  rules:
+  - name: machine-set-failure-domain
+    match:
+      resources:
+        kinds:
+        - cluster.x-k8s.io/v1alpha4/MachineSet
+        - cluster.x-k8s.io/v1beta1/MachineSet
```

Comment:
@tfussell should not be Machine Deployment?

Reply:
Good point. I should have changed the title to say MachineSet. If we default the MachineDeployment then the controller keeps trying to update it and it stops working. Defaulting the MachineSet seemed to work so I went with that.